src/backend/d3d12/CommandBufferD3D12.cpp

```diff
@@ -256,6 +256,36 @@ namespace d3d12 {
                         D3D12_RECT scissorRect = { 0, 0, static_cast<long>(width), static_cast<long>(height) };
                         commandList->RSSetViewports(1, &viewport);
                         commandList->RSSetScissorRects(1, &scissorRect);
+
+                        for (uint32_t slot = 0; slot < currentRenderPass->GetAttachmentCount(); ++slot) {
```

Comment:
The semantic here is that all of the clears in a render pass are done "before first use" in that render pass. So for opengl/metal, I clear each attachment on the subpass that first uses it, but on d3d12 I just clear everything once at the beginning. However it could be easily changed to the way I did it in opengl/metal. wdyt?